cli/provider/cmd.go

```
@@ -1048,6 +1056,9 @@ func (c *CmdConfigurator) ValidateFlags(ctx context.Context, cmd *cobra.Command)
 			}
 		}
 
+		globalPassthrough, _ := cmd.Flags().GetBool("global-passthrough")
```

don't ignore the error handling here.